tests/jerry/es.next/symbol-replace.js

```diff
@@ -682,7 +682,6 @@ Object.defineProperty(abruptStickyRegexp, 'sticky', {
 
 try {
   abruptStickyRegexp[Symbol.replace]();
-  assert(false);
 } catch (e) {
-  assert(e === "abrupt sticky");
+  assert(false);
```

Comment:
Since this should not throw error anymore, how about adding an assert for it's expected result?
`assert(abruptStickyRegexp[Symbol.replace]() === 'undefinedndefined');`